src/opera/util/metadata_utils.py

```diff
@@ -660,3 +660,150 @@ def create_test_cslc_metadata_product(file_path):
         velocity_x_dset = orbit_grp.create_dataset("velocity_x", data=np.zeros(12,), dtype='float64')
         velocity_y_dset = orbit_grp.create_dataset("velocity_y", data=np.zeros(12, ), dtype='float64')
         velocity_z_dset = orbit_grp.create_dataset("velocity_z", data=np.zeros(12, ), dtype='float64')
+
+
+def get_disp_s1_product_metadata(file_name):
+    """
+    Returns a python dict containing the DISP S1 metadata
+    which will be used with the ISO metadata template.
+
+    Parameters
+    ----------
+    file_name : str
+        the DISP S1 metadata file.
+
+    Returns
+    -------
+    disp_metadata : dict
+        python dict containing the HDF5 file metadata which is used in the
+        ISO template.
+    """
+    disp_metadata = {
+        'identification': get_hdf5_group_as_dict(file_name, f"{S1_SLC_HDF5_PREFIX}/identification"),
+    }
+
+    return disp_metadata
+
+
+def create_test_disp_metadata_product(file_path):
+    """
+    Creates a dummy DISP-S1 h5 metadata file with expected groups and datasets.
+    This function is intended for use with unit tests, but is included in this
+    module, so it will be importable from within a built container.
+
+    Parameters
+    ----------
+    file_path : str
+        Full path to write the dummy CSLC H5 metadata file to.
+
+    """
+
+    pge_runconfig_contents = """input_file_group:
```

Comment:
Including a whole runconfig might be overkill for the purposes of a test product, particularly since the runconfig field is not one we'll really care about when using the extracted metadata for the ISO xml or file naming conventions. Feel free to shorten to just a simple string, such as "dummy runconfig".